partner_multi_company/README.rst

```diff
@@ -0,0 +1,99 @@
+=====================
```

Comment:
@jokekerker welcome to OCA, thank you for your contribution.
Please update your README file with generator.